tests/Unit/Evolution/Systems/Cce/Test_ReadBoundaryDataH5.cpp

```diff
@@ -45,7 +45,8 @@ void create_fake_time_varying_modal_data(
     const gsl::not_null<Scalar<ComplexModalVector>*> dr_lapse_coefficients,
     const AnalyticSolution& solution, const double extraction_radius,
     const double amplitude, const double frequency, const double time,
-    const size_t l_max, const bool convert_to_goldberg = true) noexcept {
+    const size_t l_max, const bool convert_to_goldberg = true,
+    const bool apply_normalization_bug = false) noexcept {
```

Comment:
it applies the bug; it deliberately normalizes incorrectly so that we can check that the new utilities appropriately undo the incorrect normalization.